test/testdrive/temporal.td

```diff
@@ -0,0 +1,278 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ set-sql-timeout duration=125ms
+
+#
+# Using mz_logical_timestamp() with a VIEW source
+#
+
+> DROP VIEW IF EXISTS two_bounds_two_cols CASCADE
```

Comment:
I thought testdrive does this catalog cleanup for us at the start of each run. was there a specific reason you put these DROP statements here?

Reply:
I have no clue what this is. I copy/pasted from some other td file, and apparently screwed that up. Will fix.

Reply:
Oh sorry, this is a left-over of mine . I was debugging an issue in testdrive where if you had a dangling stand-alone space character left on its own on a line the entire flow of the test will get messed up and testdrive will attempt to run the same DDL multiple times . @frankmcsherry can I push a fix?

Reply:
Ah, no this is stuff that @philip-stoev added, that's why I had no clue what was going on. :)

Is there a good place to point Philip about testdrive `.td` "best practices"? I wouldn't know what to say about which views to create and remove.

Reply:
> @frankmcsherry can I push a fix?

Feel welcome to!